src/main/resources/com/puppycrawl/tools/checkstyle/configuration_1_4.dtd

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
> So what do we need to change? Only comments in dtd schemes?

if we change only comment - version should not be bumped.

> Do you plan to upload 1_0 - 1_X schemes to sourceforge too?

yes, all old versions on dtd should be accessible, users still works on very old versions of checkstyle.